R/neighborhood_ses.R

```diff
@@ -0,0 +1,191 @@
+#' @title
+#' Obtain commonly used neighbourhood-level socioeconomic (SES) variables
+#'
+#' @description
+#' The `neighborhood_ses()` function merges SES variables based on dissemination area ID. Function descriptions contains the necessary understanding of variable concepts and corresponding application recommendations.
+#'
+#' @section Data source
+#' All SES variables are sourced from [Statistics Canada Census.](https://www12.statcan.gc.ca/census-recensement/2021/ref/98-304/2021001/chap1-eng.cfm)
+#'
+#' The Census of Population provides a detailed and comprehensive statistical portrait of Canada. It’s collected by Statistics Canada conducts once every 5 years, aiming to provide population and dwelling counts for communities of all sizes across Canada.
+#'
+#' On-Marg is a neighborhood-level index showing marginalization differences between areas. It’s created using a portion of Statistics Canada Census variables.
+#' On-Marg has 4 dimensions. There was a name change in 2021 to avoid deficit-based language and better reflect the census measures associated with each dimension
```

Comment:
added description of each dimension, note 1: 2016 vs 2021 description seems similar, so those description may seem repetitive, but it's hard to aggregate them since variable names were changed with a legit reason. 

Note 2: There's room for argument for whether to put 2016 or 2021 first. As a counter argument: many people are using an older version of DB since their research started long ago, making them having to stick w 2016. Personally feel neutral about which version to go first.